<?xml version="1.0" encoding="UTF-8"?>
<!--
Copyright (C) 2011 David Sugar, Haakon Eriksen, GNU Free Call Foundation

This file is part of SwitchView.

SwitchView is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published
by the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

SwitchView is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with SwitchView.  If not, see <http://www.gnu.org/licenses/>.
-->

<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SwitchView Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Notify:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyConnecting">
         <property name="text">
          <string>Connecting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyDisconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyErrors">
         <property name="text">
          <string>Errors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyWarning">
         <property name="text">
          <string>Warning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyInfo">
         <property name="text">
          <string>Notices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyRegistry">
         <property name="text">
          <string>Registry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyCalls">
         <property name="text">
          <string>Calls</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Desktop:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="desktopTray">
         <property name="text">
          <string>System Tray</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="desktopMenu">
         <property name="text">
          <string>Floating Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Events:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="eventsReconnect"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Reconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
